ASM/c/dungeon_info.c

```diff
@@ -110,7 +113,7 @@ void draw_dungeon_info(z64_disp_buf_t *db) {
         int font_width = 6;
         int font_height = 11;
         int padding = 1;
-        int rows = 13;
+        int rows = (!SHUFFLE_CHEST_GAME || SHUFFLE_CHEST_GAME == 2) ? 13 : 14;
```

Comment:
Can be simplified to:

```suggestion
        int rows = SHUFFLE_CHEST_GAME == 1 ? 14 : 13;
```